mobile/lib/util/constants.dart

```diff
@@ -0,0 +1,56 @@
+// This file contains global constants.
+// Note that global constants should only be use sparingly.
+// They make sense for e.g. keys to identify widgets for testing.
+// Hint: If you want to define a color it is recommended to use extend a Theme rather than adding a constant.
+
+import 'package:flutter/material.dart';
+
+// ############################################################################
+// Common keys
+
+const _tabs = "tabs/";
+const _button = "button/";
+const _slider = "slider/";
+
+// ############################################################################
+// Main keys
+
+const _wallet = "wallet/";
+const _trade = "trade/";
+
+// path on screen
+
+const _bottomSheet = "bottom_sheet/";
+const _confirmSheet = "confirm/";
+
+// concrete selectors
```

Comment:
Are we going to have a constant for every single clickable item?

Reply:
I would make it a rule to identify widgets that we use in tests with a specific key.
I would not add them to "everything" but add them as needed when we write tests.